storage/pebble/registers.go

```diff
@@ -28,21 +28,22 @@ func NewRegisters(db *pebble.DB) (*Registers, error) {
 	registers := &Registers{
 		db: db,
 	}
-	// check height keys and populate cache. These two variables will have been set
+
+	// load first and last heights and cache the values, if the first or last height is not successfully loaded
+	// we return bootstrapped error indicating the bootstrap did not run successfully
 	firstHeight, err := registers.firstStoredHeight()
 	if err != nil {
-		// this means that the DB is either in a corrupted state or has not been initialized with a set of registers
-		// and firstHeight
-		return nil, fmt.Errorf("unable to initialize register storage, first height unavailable in db: %w", err)
+		return nil, errors.Wrap(storage.ErrNotBootstrapped, "first height not loaded")
 	}
 	latestHeight, err := registers.latestStoredHeight()
 	if err != nil {
-		// and firstHeight
-		return nil, fmt.Errorf("unable to initialize register storage, latest height unavailable in db: %w", err)
+		return nil, errors.Wrap(storage.ErrNotBootstrapped, "latest height not loaded")
 	}
-	/// All registers between firstHeight and lastHeight have been indexed
+
+	// all registers between firstHeight and lastHeight have been indexed
 	registers.firstHeight = firstHeight
 	registers.latestHeight.Store(latestHeight)
```

Comment:
Did we initialize the latestHeight?
Would be great if we never created a partial initialized registers where some fields are not created.
```suggestion
	latest := atomic.NewUint64(latestHeight)
	registers := &Registers{
		db: db,
		firstHeight: firstHeight,
		latestHeight: latest,
	}
```